### DISCUSSION TITLE
Why does the "Jobs profile" option in GitHub settings still exist?

### DISCUSSION THREAD

**Discussion by anikghosh256** [community/community#46582] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Since 2021, GitHub Jobs have been deprecated . Why does the "Jobs profile" option in GitHub settings still exist?

**Comment by jge162** [comment]
Because I think the goal was maybe to get hired, but now with so many layoffs its more unlikely now. But, you are right doesn't seem functional atm.
